src/Analyzer.JsonRuleEngine/Operators/InOperator.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.Operators
+{
+    /// <summary>
+    /// An operator that evaluates the "in" JSON expression.
+    /// </summary>
+    internal class InOperator : LeafExpressionOperator
+    {
+        /// <summary>
+        /// Gets the name of this operator.
+        /// </summary>
+        public override string Name => "In";
+
+        /// <summary>
+        /// Creates an InOperator.
+        /// </summary>
+        /// <param name="specifiedValue">The value specified in the JSON rule, it has to be an array of basic JSON values.</param>
+        /// <param name="isNegative">Whether the result of EvaluateExpression() should be negated or not.</param>
```

Comment:
For references to code (like EvaluateExpression() here), put it in a `<see cref="..."/>` tag.

Reply:
Adding it everywhere then :)